samples/sensor/wsen_hids/prj.conf

```diff
@@ -0,0 +1,16 @@
+###### Basic configuration required for using sensors ######
+CONFIG_I2C=y
+# CONFIG_SPI=y
+CONFIG_SENSOR=y
+
+###### Configuration of WSEN-HIDS ######
+CONFIG_WSEN_HIDS_TRIGGER_NONE=y
+# CONFIG_WSEN_HIDS_TRIGGER_GLOBAL_THREAD=y
```

Comment:
I know your intention but I'd rather keep the sample simple (not a blocker, no strong feeling)

Reply:
Ok, I removed all lines that are not required.